src/Components/Tittle/Tittle.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+
+interface TittleProps{
+  value:string,
+  styles:string,
+  wrapperProps:object,
```

Comment:
it is not a good solution to mark in as object. U can do it like that React.HTMLAttributes<any> or paste ur types instead of any